woocommerce/woocommerce
# Try speeding up product import unit test

### All Submissions:






### Changes proposed in this Pull Request:

The current product import unit test include multiple products with images, this PR removes most image and only include one product with images to test the image functionality. This speeds up the test as it requires less time to download images.

### How to test the changes in this Pull Request:

1. Run `vendor/bin/phpunit tests/unit-tests/importer/product.php` from within the plugin folder and make sure all tests pass.

### Other information:
